composer/swipe-composer.js

```diff
@@ -78,9 +78,9 @@ exports.SwipeComposer = Composer.specialize( /** @lends SwipeComposer# */ {
             this._startX = touch.clientX;
             this._startY = touch.clientY;
             this._startTimestamp = event.timeStamp;
-            document.addEventListener("touchmove", this, true);
-            document.addEventListener("touchend", this, true);
-            document.addEventListener("touchcancel", this, true);
+            this._element.addEventListener("touchmove", this, true);
+            this._element.addEventListener("touchend", this, true);
+            this._element.addEventListener("touchcancel", this, true);
```

Comment:
agreed for listening on the element swipe composer for unload and load but not for touchmove, touchend, and touchcancel